docs/alerts/monitors/alert-response.md

```diff
@@ -228,20 +228,90 @@ For example, the card below shows that `ServiceUnavailable` error is happening 3
 A _Monitor_ creates an _Alert_. Using the options below, you're subscribing to an _Alert's Monitor_.
 :::
 
-
 #### From your Alerts list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row, click the three-dot kebab menu > select **Subscribe**
 * Single-click on a row item > on the opened Alert page, click the three-dot kebab menu > **Subscribe to Monitor**
 
-
 #### From your Monitors list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row > click the three-dot kebab menu > click **Subscribe**
 * Single-click on a row item > in the side panel (Monitor Details), click **More Actions** > **Subscribe**
 
+#### From a folder
+
+If you subscribe from a monitor folder, then all nested monitors and folders within that folders become automatically subscribed.
+
+For example, if you create a subscription on “Monitor A”, and then move it to subscribed “Folder B”, “Monitor A” will have two subscriptions because it’s directly subscribed and inherits subscription from its parent folder.
+
+<details><summary>Click to see examples</summary>
+
+#### Example 1
+
+```bash title="Initial state"
+📁 Folder A ("No")
+├── Monitor B ("No")
+└── Monitor C ("No")
+```
+
+```bash title="Create subscription on Folder A"
+📁 Folder A ("Yes")
+├──Monitor B ("Yes (inherited from folder)")
+└──Monitor C ("Yes (inherited from folder)")
+```
+
+#### Example 2
+
+```bash title="Initial state"
+📁 Folder A ("No")
+├── Monitor B ("No")
+├── Monitor C ("No")
+└── 📁 Folder D ("No")
+    └── Monitor E ("No")
+```
+
+```bash title="Create subscription on Folder D"
+📁 Folder A ("No")
+├── Monitor B ("No")
+├── Monitor C ("No")
+└── 📁 Folder D ("Yes")
+    └── Monitor E ("Yes (inherited from folder)")
+ ```
+
+#### Example 3
+
+```bash title="Initial state"
+📁 Folder A ("No")
+├── Monitor B ("No")
+├── Monitor C ("No")
+└──  📁 Folder D ("No")
+    └── Monitor E ("Yes")
+```       
+
+```bash title="Create subscription on Folder D"
+📁 Folder A ("No")
+├── Monitor B ("No")
+├── Monitor C ("No")
+└── 📁 Folder D ("Yes")
+    └── Monitor E ("Yes")
+```       
+
+```bash title="Remove subscription on Monitor E"
+📁 Folder A ("No")
+├── Monitor B ("No")
+├── Monitor C ("No")
+└── 📁 Folder D ("Yes")
+    └── Monitor E ("Yes (inherited from folder)"
+```
+
+</details>
+
+To cancel an inherited subscription, you need to remove subscription from a parent folder or move the monitor or folder into another location outside the folder with direct subscription.
+
+Click **Unsubscribe** to learn how to remove a subscription.
+
```

Comment:
We can also add - You can choose "My subscriptions" filter on alerts list page or change the default filter by choosing `Enable "My subscriptions" as default filter` under user preferences to see all subscribed alerts on alerts list page. Worth including a screenshot of both.

Reply:
Makes sense